Source/Core/Common/Image.cpp

```diff
@@ -0,0 +1,183 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <Common/Image.h>
+
+#include <csetjmp>
+#include <cstdlib>
+#include <cstring>
+#include <memory>
+#include <png.h>
+#include <vector>
+
+#include <Common/CommonTypes.h>
+#include <Common/Logging/Log.h>
+
+namespace Common
+{
+static inline int MultiplyAlpha(int alpha, int color)
```

Comment:
Inline is likely not necessary here. `static` should already be enough. If it's not, a comment should be provided.